updating/updating-restricted-network-cluster.adoc

```diff
@@ -55,19 +57,121 @@ If you are upgrading from version 4.4.8 or later, you can use the `oc` CLI to cr
+
+* For more information on installing Operators, see xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="registry-configuration-for-update-service"]
+=== Configuring access to a secured registry for the OpenShift update service
+
+If the release images are contained in a secure registry, complete the steps in xref:../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access] along with following changes for the update service.
+
+The OpenShift Update Service Operator needs the config map key name `updateservice-registry` in the registry CA cert.
+
+.Image registry CA config map example for the update service
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-registry-ca
+data:
+  updateservice-registry: | <1>
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+  registry-with-port.example.com..5000: | <2>
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+----
+<1> The OpenShift Update Service Operator requires the config map key name updateservice-registry in the registry CA cert.
+<2>  If the registry has the port, such as `registry-with-port.example.com:5000`, `:` should be replaced with `..`.
+
+[id="update-service-install"]
+=== Installing the OpenShift Update Service Operator
+
+To install the OpenShift Update Service, you must first install the OpenShift Update Service Operator by using the {product-title} web console or CLI.
+
+[NOTE]
+====
+For clusters that are installed on restricted networks, also known as disconnected clusters, Operator Lifecycle Manager by default cannot access the Red Hat-provided OperatorHub sources hosted on remote registries because those remote sources require full internet connectivity. For more information, see xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
+====
+
+include::modules/update-service-install-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-install-cli.adoc[leveloffset=+2]
+
+include::modules/update-service-graph-data.adoc[leveloffset=+2]
+
+include::modules/update-service-mirror-release.adoc[leveloffset=+2]
+
+[id="update-service-create-service"]
+=== Creating an OpenShift Update Service application
+
+You can create an OpenShift Update Service application by using the {product-title} web console or CLI.
+
+include::modules/update-service-create-service-web-console.adoc[leveloffset=+3]
+
+include::modules/update-service-create-service-cli.adoc[leveloffset=+3]
+
+[NOTE]
+====
+The policy engine route name must not be more than 63 characters based on RFC-1123. If you see `ReconcileCompleted` status as `false`  with the reason `CreateRouteFailed` caused by `host must conform to DNS 1123 naming convention
```

Comment:
This error message seemed a little long inline. Not super important (and maybe it's fine how it is) but could maybe be reworked as a code block, something like:

```
If you see `ReconcileCompleted` status as `false`  with the reason `CreateRouteFailed` caused by:

[source,terminal]
----
host must conform to DNS 1123 naming convention and must be no more than 63 characters
----

try creating the Update Service with a shorter name.
```

... not sure if that's actually any better though since it maybe has its own problems too. :shrug: :sweat_smile: